src/uct/tcp/tcp_ep.c

```diff
@@ -591,6 +591,88 @@ static ucs_status_t uct_tcp_ep_keepalive_enable(uct_tcp_ep_t *ep)
 #endif /* UCT_TCP_EP_KEEPALIVE */
 }
 
+static ucs_status_t uct_tcp_iface_check_rp_filter(uct_tcp_iface_t *iface,
+                                                  ucs_log_level_t log_level,
+                                                  int suppress_error)
```

Comment:
suppress_erro rparam is redundant
this function can return error status anyway if failed, and the caller can decide if the error is fatal or not based on iface config